### DISCUSSION TITLE
Welcome to tw3-combat-skills Discussions!

### DISCUSSION THREAD

**Discussion by Aelto** [Aelto/tw3-combat-skills#3] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by GangcAte** [comment]
Hi. I tried installing the mod using different mod managers and script mergers but can't seem to make it work. No script merger ever finds any pending merges. It does detect the mod but shows no pending merges. No Mods menu shows up in game and the mod itself doesn't work. What might be the cause? I put the bin, dlc and Mods folders into my Witcher 3 directory.

**Reply by Aelto** [reply]
Hi. Thanks for the heads up, the release script I had setup was missing the last step to generate the mergeable scripts! I've updated the [release](https://github.com/Aelto/tw3-combat-skills/releases/tag/v2.0.2) now, if you download it again it should be working as expected.

**Reply by GangcAte** [reply]
Well, there are still no pending merges. The mods menu does appear and I can edit the mod settings. Still some things don't work, like Geralt can still miss the sidestep parry even if I have "Can miss" disabled. Also, sometimes when I use the sidestep parry, I cannot attack until I sidestep parry again a few times.

**Comment by GangcAte** [comment]
I also get no i-frames and can get killed mid-animation. About 40% of the time after I sidestep parry I cannot attack until I sitestep parry again. I must be installing something wrong. I put the bin, dlc and mods folder into my Witcher 3 game directory, there are proper files in proper places, still no scripts to merge in the Script Merger (but Script Merger itself is installed properly since I have merged the scripts of other mods). The mod itself, along with the menu, works otherwise. The inability to attack makes the game borderline unplayable, though. If you have some time to troubleshoot, I can add you on Discord. I really like this mod so I'm desperate to get it working.

**Comment by xtianray** [comment]
Thank you for the great mod. May I request that you allow us to change the damage modifier for the side-step counters? I find them too weak compared to normal light attacks. I'm also wondering if you can incorporate an increase in critical hit for the side-step counter. The idea is that the counter will be a high-risk high-reward combat move.
